nerfstudio/data/datasets/depth_dataset.py

```diff
@@ -31,14 +40,50 @@ class DepthDataset(InputDataset):
         scale_factor: The scaling factor for the dataparser outputs.
     """
 
-    exclude_batch_keys_from_device = InputDataset.exclude_batch_keys_from_device + ["depth_image"]
-
     def __init__(self, dataparser_outputs: DataparserOutputs, scale_factor: float = 1.0):
```

Comment:
Add to the class description that the depth dataset will generate data using the Zoe model if it is not found.